stylesheets/style.css

```diff
@@ -0,0 +1,639 @@
+  display: flex;
+  flex-direction: row;
+  background-color: var(--stages-subcontainers-color);
+  padding: 2% 5%;
+  gap: 25px;
+  justify-content: start;
+}
+
+.stagesTitle {
+  width: 8rem;
+  display: flex;
+  flex-direction: row;
+  gap: 15px;
+  padding-right: 5%;
+  font-size: 0.9rem;
+  color: var(--primary-theme-color);
+}
+
+.stagesImages {
+  width: 2.8rem;
+}
+
+.stagesDescription {
+  color: var(--nav-bg-color);
+}
+
+.stagesCTA {
+  margin: auto;
+  padding: 5% 6%;
+  background-color: var(--primary-theme-color);
+  margin-top: 6%;
+  box-sizing: border-box;
+}
+
+.stagesCTA:hover {
+  background-color: rgba(0, 0, 0, 0);
+  border: 2px solid var(--primary-theme-color);
+}
+
+/* FEATURED JUDGES SECTION */
+#featuredJudgesSection {
+  padding: 10% 2%;
+  color: var(--navTopbar-bg);
+}
+
+.judgesCardsContainer {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.judgesCards {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  font-weight: 900;
+  justify-content: center;
+  gap: 0;
+}
+
+.judgePatternBgs {
+  background-color: #e5e5f7;
+  background-image: radial-gradient(ellipse farthest-corner at 10px 10px, #310d29, #310d29 50%, #e5e5f7 50%);
+  background-size: 10px 10px;
+  width: 50px;
+  height: 50px;
+  top: 0;
+  left: 0;
+  border-radius: 100%;
+  opacity: 0.3;
+  z-index: -1;
+}
+
+.judgeImages {
+  margin-top: 7%;
+  margin-left: -9%;
+  padding-right: 2%;
+}
+
+.judgeContents {
+  width: 52%;
+  margin-top: 4%;
+}
+
+.judgeNames {
+  font-weight: 900;
+  font-size: 1.5rem;
+  line-height: 0;
+}
+
+.judgeIntros {
+  color: var(--primary-theme-color);
+  font-style: italic;
+  font-size: 80%;
+}
+
+.judgeDividers {
+  width: 10%;
+  margin-left: 0;
+  border-bottom: var(--navTopbar-bg);
+}
+
+.judgeDescriptions {
+  font-size: 80%;
+  box-sizing: border-box;
+  font-weight: 700;
+}
+
+.loadMore {
+  border: 2px solid var(--primary-font-color);
+  padding: 2%;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  font-weight: 700;
+}
+
+.loadMore:hover {
+  background-color: var(--primary-theme-color);
+  border: none;
+  color: var(--nav-bg-color);
+}
+
+/* PARTNERS SECTION */
+.partnerSection {
+  background-color: var(--navTopbar-bg);
+  display: flex;
+  flex-direction: column;
+  padding: 15% 3%;
+  gap: 5px;
+  z-index: -9;
+  position: relative;
+}
+
+.logoContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  gap: 18px;
+  flex-wrap: wrap;
+}
+
+.partnersSectionTitle {
+  color: var(--nav-bg-color);
+}
+
+.partners {
+  filter: grayscale(-1);
+  -webkit-filter: grayscale(100%);
+  position: static;
+  z-index: -1;
+}
+
+.partners:hover {
+  filter: none;
+  -webkit-filter: none;
+}
+
+/* FOOTER SECTION */
+footer {
+  display: flex;
+  flex-direction: row;
+  padding: 3% 2%;
+  justify-content: center;
+  font-size: 0.9rem;
+  font-weight: 400;
+  gap: 8px;
+  background-color: var(--lightgray);
+}
```

Comment:
- [OPTIONAL] KIndly make sure the footer contents are aligned correctly in the center of the footer section in order to match the provided Behance design. Kindly remember to fix this issue on both pages.

   - ### YOUR DESIGN
   ![image](https://github.com/geekelo/gft-hackathon-webapp-project/assets/87197412/2897c88f-a80c-4088-bdd7-54532b492b6f)

   ![image](https://github.com/geekelo/gft-hackathon-webapp-project/assets/87197412/209dfbe2-c212-43a7-9fba-a954aeab00b8)


   - ### BEHANCE DESIGN
   ![image](https://github.com/geekelo/gft-hackathon-webapp-project/assets/87197412/acb5f38e-f5c1-4451-9bf7-8371e1fc4811)

  ![image](https://github.com/geekelo/gft-hackathon-webapp-project/assets/87197412/42ddf135-38f6-4d7b-b3d2-8d3de2dded75)

Reply:
@levy002 Thanks for the review
I would like to update you that the footer contents are now aligned correctly in the center of the footer section in order to match the provided Behance design.